cobigen-templates/templates-oasp4j/src/main/templates/crud_java_server_app/functions.ftl

```diff
@@ -92,6 +92,9 @@
 		}</#if>
 	</#if>
 <#elseif !isSearchCriteria || !JavaUtil.isCollection(classObject, field.name)>
+  /**
+   * @return ${field.name}Id
+   */
```

Comment:
we don't need javadoc on getters, which have @override annotations. Please check that.